com.unity.render-pipelines.universal/Shaders/Utils/Universal2D.hlsl

```diff
@@ -4,13 +4,13 @@
 struct Attributes
 {
     float4 positionOS       : POSITION;
-    float2 uv               : TEXCOORD0;
+    half2  uv               : TEXCOORD0;
 };
 
 struct Varyings
 {
-    float2 uv        : TEXCOORD0;
-    float4 vertex : SV_POSITION;
+    float4 vertex    : SV_POSITION;
+    half2  uv        : TEXCOORD0;
```

Comment:
Careful with this, depending on your texture resolution you won't have enough precision to store uv coord in half precision.